### PR TITLE
Add request.id

### PR DESCRIPTION
**Pull request by allevo**
As titled. I like to have `request.id` as alias `request.raw.id`

Do you agree?

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

~~I don't know where to test it.
I'll update the docs asap.~~

I add the typescript definition too.

### CODE REVIEW

File: lib/request.js [hunk 1]
```diff
@@ -29,5 +29,11 @@ Object.defineProperty(Request.prototype, 'req', {
   }
 })
 
+Object.defineProperty(Request.prototype, 'id', {
```
Comment (StarpTech) [inline lib/request.js]: Combine both with `defineProperties`

Comment (allevo) [reply lib/request.js]: done

File: docs/Request.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ Request is a core Fastify object containing the following fields:
 - `params` - the params matching the URL
 - `headers` - the headers
 - `raw` - the incoming HTTP request from Node core *(you can use the alias `req`)*
+- `id` - the id generated for this request
```
Comment (StarpTech) [inline docs/Request.md]: I would say "the request id"

Comment (allevo) [reply docs/Request.md]: done

**Pull request merged**